lib/utils.js

```diff
@@ -127,11 +127,51 @@ const colors = {
   green: (message) => `\u001b[1m\u001b[32m${message}\u001b[39m\u001b[22m`
 }
 
+/**
+ * @param {String} packagePath full path to package.json
+ * @param {String} appName name of the app
+ * @description Updates the name property in JSON file
+ * @returns {Promise} 
+ */
+ function updatePackageJson(packagePath, appName) {
+  const packageJSON = require(packagePath);
+  packageJSON.name = appName;
+  return new Promise((resolve, reject) => {
+    fs.writeFile(packagePath, JSON.stringify(packageJSON, null, 2), function(err) {
+      if (err) reject(err);
+      resolve();
+    });
+  });
+}
+
+/**
+ * @param {String} appPath path of the app
+ * @description Creates a list of json files to be modified and invokes the updateFN parllely 
+ * @returns {Promise} 
+ */
+async function setCorrectAppName(appPath) {
+  try {
+    const currentDirectory = process.cwd();
+    const appName = appPath.split(path.sep).pop();
```

Comment:
We can replace this with,
```suggestion
    const appName = path.basename(appPath)
```
This will be more readable and reliable I guess

Reply:
Sure, this looks more cleaner and readable